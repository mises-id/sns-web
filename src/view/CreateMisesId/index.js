/*
 * @Author: lmk
 * @Date: 2021-07-15 10:45:52
 * @LastEditTime: 2021-07-15 16:42:08
 * @LastEditors: lmk
 * @Description: create mises page
 */
import './index.scss';
import React from 'react';
import { useTranslation } from 'react-i18next';
import { Button } from 'zarm';
const list = [1,2,2,4,2,2,2,3,2,2] //test
const CreateMisesId = (props)=>{
  const {t} = useTranslation()
  return <div className="m-flex m-flex-col m-layout">
    <div className="m-flex-1 m-padding15" style={{height:0,overflowY:'auto'}}>
      <p className="m-title">{t('aboutId')}</p>
      {list.map((val,index)=>(<p key={index} className="m-font15 m-margin-top-8">This is the introduction. This is the introduction. This 
is the introduction. This is the introduction. This is the
 introduction. This is the introduction. </p>))}
    </div>
    <div className="footer m-bg-fff m-text-center">
      <Button block shape="round" theme="primary" ghost>{t('createId')}</Button>
      <p className="m-padding-tb10 m-colors-666 m-font12">{t("restoreTips")}</p>
      <p className="m-font15 m-colors-5c65f6">{t("restore")}</p>
    </div>
  </div>
}
export default CreateMisesId