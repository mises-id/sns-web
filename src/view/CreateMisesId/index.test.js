/*
 * @Author: lmk
 * @Date: 2021-07-15 10:46:27
 * @LastEditTime: 2021-07-15 10:47:10
 * @LastEditors: lmk
 * @Description: CreateMisesIdtest
 */
import React from 'react';
import {shallow } from 'enzyme';
import CreateMisesId from './';
describe('CreateMisesId', () => {
  it('should render correctly', () => {
    const component = shallow(<CreateMisesId />)
    expect(component).toMatchSnapshot();
  });
});