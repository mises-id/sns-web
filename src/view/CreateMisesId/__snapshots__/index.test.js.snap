// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateMisesId should render correctly 1`] = `
<div
  className="m-flex m-flex-col m-layout"
>
  <div
    className="m-flex-1 m-padding15"
    style={
      Object {
        "height": 0,
        "overflowY": "auto",
      }
    }
  >
    <p
      className="m-title"
    >
      aboutId
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="0"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="1"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="2"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="3"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="4"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="5"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="6"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="7"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="8"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
    <p
      className="m-font15 m-margin-top-8"
      key="9"
    >
      This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. This is the introduction. 
    </p>
  </div>
  <div
    className="footer m-bg-fff m-text-center"
  >
    <Button
      block={true}
      disabled={false}
      ghost={true}
      htmlType="button"
      loading={false}
      onClick={[Function]}
      prefixCls="za-button"
      shadow={false}
      shape="round"
      size="md"
      theme="primary"
    >
      createId
    </Button>
    <p
      className="m-padding-tb10 m-colors-666 m-font12"
    >
      restoreTips
    </p>
    <p
      className="m-font15 m-colors-5c65f6"
      onClick={[Function]}
    >
      restore
    </p>
  </div>
</div>
`;
